packages/jss-starter-kit/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "jss-starter-kit",
```

Comment:
still thinking about the name, other optitons:

- jss-demo-kit/bundle
- jss-playground+-(bundle/kit)

Reply:
"starter kit" and "playground" are both known concepts in English.  I wouldn't mix them, but if you prefer `playground` to `starter-kit`, I can make the change.

Reply:
I was just brainstorming, if tou think its the best name, lets go for it

Reply:
Cool.  I don't have strong feelings on it.  `jss-starter-kit` and `jss-playground` are both fine for me.